src/includes/head.pug

```diff
@@ -6,20 +6,16 @@ head
   link(rel='shortcut icon' href='assets/images/favicon.png' type='image/png')
   link(rel='icon' href='assets/images/favicon.png' type='image/png')
   link(rel='apple-touch-icon' href='assets/images/apple-touch.png')
-  link(rel='apple-touch-icon' sizes='72x72' href='assets/images/apple-touch-72.png')
-  link(rel='apple-touch-icon' sizes='114x114' href='assets/images/apple-touch-114.png')
-  link(rel='apple-touch-icon' sizes='144x144' href='assets/images/apple-touch-144.png')
-  
+
```

Comment:
any reason for ditching these icons? just wondering

Reply:
According to Sonarwhal's advice it's redundant to specify all the different sizes - just one, high-res icon will suffice. See https://sonarwhal.com/docs/user-guide/rules/apple-touch-icons/#page-heading That makes sense to me, so I removed them.